sup3r/preprocessing/data_handling.py

```diff
@@ -264,11 +264,15 @@ def file_paths(self, file_paths):
         """
         self._file_paths = file_paths
         if isinstance(self._file_paths, str):
-            if '*' in self._file_paths:
+            if '*' in file_paths:
                 self._file_paths = glob.glob(self._file_paths)
             else:
                 self._file_paths = [self._file_paths]
 
+        msg = ('No valid files provided to DataHandler. '
+               f'Received file_paths={file_paths}. Aborting.')
+        assert len(self._file_paths) > 0 and file_paths is not None, msg
```

Comment:
Probably better to check the None condition first. If file_paths is None, you'll get a weird error on cannot run `len(None)` instead of `is not None` i think?

Reply:
Yeah good call